tools/walletextension/api/staticOG/javascript.js

```diff
@@ -39,6 +39,7 @@ async function addNetworkToMetaMask(ethereum, userID, chainIDDecimal) {
                 {
                     chainId: chainIdHex,
                     chainName: 'Obscuro Testnet',
+                    iconUrls: [https://raw.githubusercontent.com/obscuronet/go-obscuro/main/tools/walletextension/api/staticOG/Metamask%20Network%20Icon.png],
```

Comment:
This approach does not work.
See the official Metamask documentation: https://docs.metamask.io/wallet/reference/rpc-api/#wallet_addethereumchain
(parameter is currently ignored)